test/bimap_properties_test.exs

```diff
@@ -54,8 +54,8 @@ defmodule BiMapPropertiesTest do
   end
 
   property "it puts items into bimaps" do
-    check all key_set <- nonempty(uniq_list_of(term())),
-              value_set <- uniq_list_of(term(), length: Enum.count(key_set)),
+    check all key_set <- nonempty(uniq_list_of(atom(:alphanumeric))),
+              value_set <- uniq_list_of(integer(), length: Enum.count(key_set)),
```

Comment:
Actually, there is a deeper problem with this test:

```elixir
iex> regular_map = %{x: 0}
iex> bimap = BiMap.new(regular_map)

iex> Map.put(regular_map, :a, 0)
#BiMap<[x: 0]>
iex> BiMap.put(bimap, :a, 0)
#BiMap<[a: 0]>
```

I'll push proper fix directly onto master